src/Microsoft.OData.Core/ODataSimplifiedOptions.cs

```diff
@@ -11,149 +11,37 @@ namespace Microsoft.OData
     /// <summary>
     /// Options which used to control the behaviour related odata simplified.
     /// </summary>
-    public sealed class ODataSimplifiedOptions
+    internal sealed class ODataSimplifiedOptionsDeleted

```

Comment:
What is happening here? I'd have expected this file to be deleted. What's the use case for retaining it?

Reply:
I thought I deleted this. My bad.

Reply:
I've deleted it.